scss/detail.scss

```diff
@@ -39,6 +46,29 @@
 }
 
 .book-cover {
+  position: relative;
   height: 368px;
-  width: 261px;
+  max-width: 400px;
+  width: 100%;
+}
+
+.book-separator {
+  border-bottom: 4px solid $earls-green;
+}
+
+.book-image-container {
+  &::after {
+  -ms-transform: rotate(15deg);
+  -webkit-transform: rotate(15deg);
+  background: url('../assets/badge.png') no-repeat;
+  background-size: 100%;
+  bottom: calc(100% - 115px);
+  content: '';
+  transform: rotate(15deg);
+  height: 110px;
+  left: calc(100% - 1050px);
```

Comment:
Oh my! Not what I had in mind. You really need to set such a big value for it to work ? 1050px ? Don't change this, just know that if it is too big of a value then it's probably best to use the opposite property: right